org.sonarlint.eclipse.ui/src/org/sonarlint/eclipse/ui/internal/Messages.java

```diff
@@ -88,6 +88,8 @@
   public static String ServerLocationWizardPage_label_password;
   public static String ServerLocationWizardPage_action_test;
   public static String ServerLocationWizardPage_action_test_tooltip;
+  public static String ServerLocationWizardPage_action_token;
+  public static String ServerLocationWizardPage_action_token_tooltip;
```

Comment:
![MINOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-minor.png) Rename this field "ServerLocationWizardPage_action_token_tooltip" to match the regular expression '^[a-z][a-zA-Z0-9]*$'. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](https://dory.sonarsource.com/sonarqube/coding_rules#rule_key=squid%3AS3008)